hal/src/stm32f2xx/bootloader_platform_10.c

```diff
@@ -1,37 +1,37 @@
 const unsigned char bootloader_platform_10_bin[] = {
```

Comment:
If a new bootloader binary is bundled here, it should probably have a bumped module version, otherwise it is not automatically upgraded, which causes a device to enter an endless loop, as well as DCT corruption.